projects/convenience.md

```diff
@@ -0,0 +1,74 @@
+# Convenience Project
+
+## Background and description
+
+Convenience as a SIG was initially proposed by Ted Young back in 2021. It
+was postponed to prioritize completion of signal specs and stabilization. A
+document was written at this time describing user research to do and a
+brainstorming session,
+https://docs.google.com/document/d/1bjiw5L4E8FztRtfsy4Nu9A6qSP-BJlK6BBzl6Rea-Ds/edit#heading=h.b7qckyjgu28i
+
+### Current challenges
+
+Currently the various language implementations resolve user convenience
+individually and this can lead to drift between different implementations and
+with the spec. An example would be how to handle marking a code block as
+`untraced` which has implementations in at least Java, Ruby and Javascript, each
+done differently, named differently and with no specification.
+
+### Goals, objectives, and requirements
+
+With the 3 main signals going stable the time is right to circle back and
+address issues users may have faced over the years they've now been using
+OpenTelemetry.
+
+This project aims to identify developer experience issues with using
+OpenTelemetry, to select 3 that are concerns which can be addressed through
+updates to the specification and resolve them.
```

Comment:
I assume this is the initial scope and that this SIG will continue afterwards?

Reply:
That makes sense, so the SIG identifies those 3 items, works on them and after they are accomplished the members of that SIG can decide if they want to continue. I like that 👍